### PR TITLE
Beta to Preview

### LINKED ISSUE #1115: rename rel/1.0.0 beta to preview
We are not yet stable enough for a beta release, but we do want to have a baseline build folks can experiment with [signed, etc].

### PR DESCRIPTION
**Pull request by piotrpMSFT**
fixes #1115

---
This change is [Reviewable](https://reviewable.io/reviews/dotnet/cli/1116)

### CODE REVIEW

File: Documentation/beta-install-experience.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-Beta install options and experience
```
Comment (livarcocc) [inline Documentation/beta-install-experience.md]: Change the name of the doc to preview-install-experience?

**Comment by livarcocc** [comment]
LGTM, expect it fails CI. I don't think you can pull from preview just it for bootstrapping.

**Comment by piotrpMSFT** [comment]
@Sridhar-MS can we copy the beta packages to a preview channel to bootstrap this?

**Comment by Sridhar-MS** [comment]
Yup. Fixing it now.

**Comment by Sridhar-MS** [comment]
@dotnet-bot test this please

**Comment by anurse** [comment]
In conflict, still relevant?

**Comment by blackdwarf** [comment]
@piotrpMSFT should we close this?

**Comment by joshfree** [comment]
moving to the 1.0.0-rc2 milestone

**Pull request closed without merging**